components/director/internal/domain/runtime/repository_test.go

```diff
@@ -49,42 +52,101 @@ func TestPgRepository_GetByID_ShouldReturnRuntimeModelForRuntimeEntity(t *testin
 	assert.Equal(t, tenantID, modelRuntime.Tenant)
 }
 
-func TestPgRepository_List_ShouldReturnRuntimeModelsForRuntimeEntities(t *testing.T) {
-	// given
-	runtime1ID := uuid.New().String()
-	runtime2ID := uuid.New().String()
-	tenantID := uuid.New().String()
-
+func TestPgRepository_List(t *testing.T) {
+	//GIVEN
 	timestamp, err := time.Parse(time.RFC3339, "2002-10-02T10:00:00-05:00")
 	require.NoError(t, err)
 
-	sqlxDB, sqlMock := mockDatabase(t)
-
-	rows := sqlmock.NewRows([]string{"id", "tenant_id", "name", "description", "status_condition", "status_timestamp", "auth"}).
-		AddRow(runtime1ID, tenantID, "Runtime ABC", "Description for runtime ABC", "INITIAL", timestamp, agentAuthStr).
-		AddRow(runtime2ID, tenantID, "Runtime XYZ", "Description for runtime XYZ", "INITIAL", timestamp, agentAuthStr)
-
-	sqlMock.ExpectQuery(`^SELECT (.+) FROM "public"."runtimes" WHERE "tenant_id" = \$1$`).
-		WithArgs(tenantID).
-		WillReturnRows(rows)
-
-	ctx := persistence.SaveToContext(context.TODO(), sqlxDB)
-
-	pgRepository := runtime.NewPostgresRepository()
-
-	// when
-	modelRuntimePage, err := pgRepository.List(ctx, tenantID, nil, nil, nil)
-
-	//then
-	assert.NoError(t, err)
-	assert.Equal(t, 2, modelRuntimePage.TotalCount)
-	require.NoError(t, sqlMock.ExpectationsWereMet())
+	tenantID := uuid.New().String()
+	runtime1ID := uuid.New().String()
+	runtime2ID := uuid.New().String()
 
-	assert.Equal(t, runtime1ID, modelRuntimePage.Data[0].ID)
-	assert.Equal(t, tenantID, modelRuntimePage.Data[0].Tenant)
+	limit := 2
+	offset := 3
+
+	pageableQuery := `^SELECT (.+) FROM "public"."runtimes" WHERE "tenant_id" = \$1 ORDER BY "id" LIMIT %d OFFSET %d$`
```

Comment:
Don't escape, use the same helper like in Labels or LabelDefinition, which will escape the characters automatically.

Reply:
Fixed in `COUNT (*)`.